### DISCUSSION TITLE
Typescript sample files not getting packaged as is

### DISCUSSION THREAD

**Discussion by automartin5000** [projen/projen#3189] [Q&A] [answered]
I'm trying to add some sample .ts files to my project, but every release I run is compiling them to .js files and not packaging the original .ts files even though I've excluded the directory in tsconfig and eslint via projenrc. I'm using the `SampleDir` and `SampleFile` components.

When I went to look at other examples, everyone else seems to be using inline content which seems...hard to manage. I'd really prefer to manage regular files and simply add them in the package.

**Comment by mrgrain** [comment, marked as answer]
Just don't put them into `src`. The `src` directory comes with a bunch of assumptions and you are finding it hard to reverse engineer all of them.

 I usually have this kind of stuff inside `examples`.

**Reply by automartin5000** [reply]
@mrgrain Got it thanks! I'll try that

**Comment by automartin5000** [comment]
> Just don't put them into `src`. The `src` directory comes with a bunch of assumptions and you are finding it hard to reverse engineer all of them.
> 
> I usually have this kind of stuff inside `examples`.

That worked, thanks!
